### PR TITLE
chore: refactor decorators file structure

### PR DESCRIPTION
**Pull request by andriyl**
## Check yourself
- [x] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security
- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

**Review by adamaltman — COMMENTED** [review]
I love this change 🎉

File: packages/core/src/config/config.ts [hunk 1]
```diff
@@ -623,3 +624,12 @@ function assignExisting<T>(target: Record<string, T>, obj: Record<string, T>) {
     }
   }
 }
+
+export function makeConfig(rules: Record<string, RuleConfig>, decorators?:  Record<string, DecoratorConfig>) {
```
Comment (RomanHotsiy) [inline packages/core/src/config/config.ts:628]: Keep this function in __tests__ helpers. We don't need this function anywhere in non-test code so why have it here.

**Review by RomanHotsiy — CHANGES_REQUESTED** [review]
Looks great, just move the test helper function `makeConfig` back to __tests__ folder

File: packages/core/src/config/config.ts (hunk 1 above)
Comment (andriyl) [reply packages/core/src/config/config.ts]: @RomanHotsiy moved to `core/__test__/utils`

**Review by RomanHotsiy — APPROVED** [review]

**Pull request merged**